ADA/ada.qmd

```diff
@@ -19,57 +17,194 @@ It brings together:
+
+To access your data and make use of cloud computing resource, you will need access to a cluster, which is a group of virtual machines that complete your processing for you. Your cluster setup will depend on which programming language you wish to use. You will be given access to a shared cluster when your data access is granted. Please see the [how clusters work](https://analytical-data-access.education.gov.uk/get-support/how-clusters-work) page on the ADA guidance site for more detail. 
+
+---
+
+#### Writing and running code
+
+---
+
+In the Databricks interface, you have the choice to use notebooks, scripts or the Databricks SQL Editor. 
+
+Notebooks are made up of multiple code or markdown cells. They are different to IDEs (Integrated Development Environments) like RStudio. You can read more about the differences between IDEs and notebooks and where each might be useful [on the NHS-R website](/ADA/ada.html#differences-between-ides-and-notebooks).
+
+Scripts (e.g. R scripts) can be created and saved in your Workspace and run via the Databricks interface. 
+
+There is detailed guidance on using the Databricks SQL editor, which is similar to SSMS, on the [ADA guidance site](https://analytical-data-access.education.gov.uk/get-support/using-sql-in-databricks-sql-editor).
+
+One benefit of Databricks is the use of workflows to run your code. Workflows allow you to automate the running of single or multiple scripts, so if you have large amounts of data processing to do on a regular basis, you can set up a workflow to complete this overnight. Your laptop does not need to be switched on for this to work. 
+
+You can also run code from RStudio or other IDEs locally and execute it against the data held in Databricks. To do this, you will need to set up a connection between RStudio and the Databricks SQL Warehouse. This is explained in the [R scripts in RStudio section](/ADA/ada.html#r-scripts-in-rstudio) below.
+
+---
+
+#### Accessing code from a git repository
+
+Databricks has inbuilt git functionality, and you can save your notebooks and scripts inside git repositories as with any other file formats. There is guidance on using Databricks with Azure DevOps on our [Databricks and version control page](ADA/git_databricks.html). 
+
+---
+
+## What this means for existing code
+
+### SQL scripts
+
+---
+
+#### Scripts currently used in SSMS
+
+If you currently use data and scripts stored in SSMS for your statistics publication pipeline, there will be small changes to your production pipelines that vary depending on your current process. 
```

Comment:
are we happy this is stats production specific? You could say for your analysis or statistics publication pipeline

Reply:
Agree - don't think we need to specify stats publication pipeline here, it's just any analysis using SQL scripts in SMSS

Reply:
This page/section was originally just about stats production but agree it's better covering things more widely

Reply:
I didn't realise it was stats specific before, I think on the analysts guide we always have to assume it could be read by anyone and be really explicit if we're making it stats specific (or put the page in the stats production section)

For the text in this comment thread, it's a bit nitpicky but I'd change

> If you currently use data and scripts stored in SSMS for your analysis or statistics publication pipeline, there will be small changes to your production pipelines that vary depending on your current process.

to

> If you currently use data and scripts stored in SSMS for your analysis, there will be small changes you need to make that vary depending on your current process.

Reply:
Don't think this has been changed yet?